RecoEgamma/EgammaIsolationAlgos/interface/EleTkIsolFromCands.h

```diff
@@ -100,40 +103,37 @@ class EleTkIsolFromCands {
   // preselected by the cuts that can already be applied without considering
   // the electron. Note that this has to be done twice, because the required
   // preselection is different for barrel and endcap electrons.
-  //
-  class SimpleTrack {
-  public:
-    SimpleTrack(reco::TrackBase const& trk) : pt{trk.pt()}, eta{trk.eta()}, phi{trk.phi()}, vz{trk.vz()} {}
-    const double pt;
-    const double eta;
-    const double phi;
-    const double vz;
-  };
+
+  SOA_DECLARE_COLUMN(Pt, double, "pt");
```

Comment:
I would suggest putting this within a namespace.

Reply:
Okay! Then I can't put it in the class, but that's fine because I plan to reuse it in other egamma code anyway. If there are no objections, I'd put it in the namespace `egamma::tracktable`.